python/paddle/fluid/layers/control_flow.py

```diff
@@ -295,11 +295,13 @@ class StaticRNN(object):
     """
     StaticRNN class.
 
-    The StaticRNN can process a batch of sequence data. The length of each
-    sample sequence must be equal. The StaticRNN will have its own parameters
-    like inputs, outputs, memories. **Note that the first dimension of inputs
-    represents sequence length, and all the sequence length of inputs must be
-    the same. And the meaning of each axis of input and output are the same.**
+    The StaticRNN can process a batch of sequence data. The first dimension of inputs
+    represents sequence length, the length of each input sequence must be equal.
+    StaticRNN will unfold sequence into time steps, use need to define how to process
+    each time step during the :code:`with` step.
+
+    Args:
+        name (str, optional): Please refer to :ref:`api_guide_Name`, Default None.
```

Comment:
Do you have the document screenshot? I'm  not sure, whether :ref:`api_guide_Name` here can link to the reference page.

Reply:
OK will add preview later

Reply:
Great